tests/unit/type/zset.tcl

```diff
@@ -78,6 +78,46 @@ start_server {tags {"zset"}} {
             assert {[r zscore ztmp y] == 21}
         }
 
+        test "ZADD GX updates existing elements when new scores are greater" {
+            r del ztmp
+            r zadd ztmp 10 x 20 y 30 z
+            r zadd ztmp gx 5 foo 11 x 21 y 29 z
```

Comment:
how about using `CH` to check the `zadd` response?

Reply:
Good suggestion

Reply:
I've added CH to several of the tests to ensure that the count of changes comes back as expected.